Header header
int8 mode
geometry_msgs/Pose pose_global
geometry_msgs/Vector3 pose_image
geometry_msgs/Vector3 angle
